benchmark/benchmark_test.go

```diff
@@ -0,0 +1,55 @@
+package benchmark
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/Layr-Labs/eigenda-proxy/clients/standard_client"
+	"github.com/Layr-Labs/eigenda-proxy/testutils"
+)
+
+// BenchmarkPutsWithSecondaryV1  ... Takes in an async worker count and profiles blob insertions using
+// constant blob sizes in parallel. Exercises V1 code pathways
+func BenchmarkPutsWithSecondaryV1(b *testing.B) {
+	testCfg := testutils.NewTestConfig(true, false)
+	putsWithSecondary(b, testCfg)
+}
+
+// BenchmarkPutsWithSecondaryV2  ... Takes in an async worker count and profiles blob insertions using
+// constant blob sizes in parallel. Exercises V2 code pathways
+func BenchmarkPutsWithSecondaryV2(b *testing.B) {
+	testCfg := testutils.NewTestConfig(true, true)
```

Comment:
can you change this constructor to use functional options like I did for the fuzzer? Makes it much easier to read what exactly is the option being passed I find.